src/tests/ftest/deployment/basic_checkout.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from ior_test_base import IorTestBase
+from mdtest_test_base import MdtestBase
+from data_mover_test_base import DataMoverTestBase
+from command_utils_base import CommandFailure
+
+class BasicCheckout(IorTestBase, MdtestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description: Test class wrapping up tests from four
+                               different test classes into one. Intent
+                               is to run these tests as a basic checkout
+                               for newly installed servers.
+    :avocado: recursive
+    """
+
+    def test_basic_checkout(self):
+        """
+        Test Description: Bundles four tests into one and run in the
+                          following sequence - ior_small, mdtest_small,
+                          ec_smoke and autotest.
+        :avocado: tags=installation,basiccheckout
```

Comment:
Tags usually look like:
```
:avocado: tags=all,daily_regression # frequency
:avocado: tags=hw,small # hardware/vm/stage
:avocado: tags=datamover # features
:avocado: tags=test_class_name,test_method_name # specific test file/method tags
```
Keeping with that spirit, could we do something like this?
```suggestion
        :avocado: tags=installation #frequency
        :avocado: tags=basiccheckout
```
Also, are we making a distinction between "installation" and "deployment"? If not, I wonder if we should use JUST one term or the other. I.e. this uses "installation" tags for the "deployment" directory.

I'm also still not sure if we should use "deployment_basiccheckout", etc. to make sure tags don't overlap in the future (unlikely here). Anyway, I have to design the tag hierarchy for https://daosio.atlassian.net/browse/DAOS-8993, so we don't have to get caught up on tags just yet as long as they don't introduce conflicts with CI (which this does not).

Reply:
Yeah these are older version which I pushed originally. They need to change to what you and I discussed the other day on Teams on-site channel. Same for 'installation', this needs to change to 'deployment'.

Reply:
Done